### ISSUE TITLE
Social media sharing pills misaligned in sidebar

### ISSUE THREAD

**Issue by gauravano** [publiclab/plots2#8431]
### Please describe the problem (or idea)

> What did you expect to see that you didn't?

Social media sharing pills not aligned in the sidebar.

<img width="318" alt="tagss" src="https://user-images.githubusercontent.com/20878070/94366279-c447eb80-00f4-11eb-976d-ea25fa97a6a5.png">

(Issue observed in https://github.com/publiclab/plots2/issues/5005)

### Please show us where to look

Observed in various pages. Example: https://publiclab.org/wiki/name-your-project-here

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site. 

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

**Comment by avats-dev** [comment]
@gauravano I don't know how but social pills alignment in https://publiclab.org/wiki/name-your-project-here works fine for me, as you can see in this gif.

![aligncheck](https://user-images.githubusercontent.com/35889327/94375981-2b859000-0135-11eb-8fff-adb4c7a07c8a.gif)

**Maybe this is platform dependent, but I'm not sure. FYI I'm using Mozilla Firefox x64 (80.0.1) on Ubuntu (20.04 LTS).**

**Comment by gauravano** [comment]
Hmm, it seems to be browser-related. I am getting correct alignment in Safari as well but in Chrome it's misaligned. I am also wondering if we can fit all the 3 pills in a single row? Currently, it's coming in 2 rows irrespective of screen size.

Thank you @avats-dev for checking it for Firefox and highlighting on browser-specific point :)

**Comment by ebarry** [comment]
This is tricky! I just went to the wiki page you linked to in Chrome, and i am not seeing the mis-alignment. Here's what i see at all screenwidths: 
<img width="183" alt="Screen Shot 2020-09-28 at 4 29 38 PM" src="https://user-images.githubusercontent.com/161439/94482838-e77fa300-01a7-11eb-8041-c622d2c82cf8.png">

**Comment by avats-dev** [comment]
:confused: 
@gauravano yeah, keeping all three in a single row might be good or keeping only one in a row :thinking:

**Comment by cesswairimu** [comment]
Yeah looks fine on my end too

**Comment by gauravano** [comment]
That's weird. Let's ignore the alignment issue then. Maybe something wrong with my browser :P. @avats-dev if you want you can place all 3 buttons in a single row. I think, separate row for each button would not look good and take more space too. What do you think @ebarry @cesswairimu?

**Comment by theBstar** [comment]
I can see the issue occurring on Chrome 85. I can raise the fix for this. Can you please assign the issue to me?

**Comment by avats-dev** [comment]
@theBstar you're good to go. :+1:  Can you also shift them all in a single row and check if it better than this? Thanks.

> We don't assign issues , and it is not needed in general.

**Comment by cesswairimu** [comment]
Yeah I agree @gauravano if they could fit on a single row that would be super. @theBstar I have assigned you the issue. Thanks

**Comment by ebarry** [comment]
Can someone check if this changes based on if they are viewing the website in another language? I now see that i'm experiencing the same bug that Gaurav highlighted in my browser that is set to a different language. @jywarren — i think this is maybe related to the other translation stuff.

**Comment by cesswairimu** [comment]
oh yeah, I can see it when I change the language

**Comment by anirudhu-ui** [comment]
Hi, I’d like to work on this issue. Could you please assign it to me?
